account_move_locking/tests/test_move_locking.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# © 2015-2016 Camptocamp SA (Vincent Renaville)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import datetime
+from openerp.tests import common
+from openerp import fields, tools
+from openerp.modules import get_module_resource
+from openerp.exceptions import UserError
+
+
+class TestMoveLocking(common.TransactionCase):
+
+    def setUp(self):
+        super(TestMoveLocking, self).setUp()
+        self.company_a = self.browse_ref('base.main_company')
+        tools.convert_file(self.cr, 'account',
+                           get_module_resource('account', 'test',
+                                               'account_minimal_test.xml'),
+                           {}, 'init', False, 'test')
+        self.account_move_obj = self.env["account.move"]
+        self.account_move_line_obj = self.env["account.move.line"]
+        self.wizard_obj = self.env["lock.account.move"]
+        self.account_id = self.ref("account.a_recv")
+        self.account_id2 = self.ref("account.a_expense")
+        self.journal_id = self.ref("account.bank_journal")
+        self.partner = self.browse_ref("base.res_partner_12")
+
+    def test_move_lock(self):
+
+        # Create entry for today
+        self.move = self.account_move_obj.create({
+            "date": fields.Date.today(),
+            "journal_id": self.journal_id
+        })
+
+        self.account_move_line_obj.with_context(
+            check_move_validity=False
```

Comment:
FYI, another way to create moves for testing without having to play with `check_move_validity`.
https://github.com/oca-sorrento/account-financial-reporting/blob/f0b2a6b56393928ce85f12043d34a909f97f1734/mis_builder/tests/test_aep.py#L87